apps/api/src/app/events/usecases/verify-payload/verify-payload.usecase.ts

```diff
@@ -0,0 +1,81 @@
+import { ITemplateVariable } from '@novu/dal';
+import { ApiException } from '../../../shared/exceptions/api.exception';
+import { VerifyPayloadCommand } from './verify-payload.command';
+
+export class VerifyPayload {
+  execute(command: VerifyPayloadCommand): Record<string, unknown> {
```

Comment:
Could we create a unit tests for this function to make sure all edge cases are covered here?